packages/wrangler/src/dev/start-server.ts

```diff
@@ -224,6 +226,7 @@ async function runEsbuild({
 	workerDefinitions: WorkerRegistry;
 	firstPartyWorkerDevFacade: boolean | undefined;
 	testScheduled?: boolean;
+	experimentalLocal: boolean | undefined;
```

Comment:
```suggestion
	experimentalLocal?: boolean;
```

Reply:
Sure (this seems like a wider thing to solve across the codebase, not as part of this PR)